packages/formstr-sdk/src/utils/utils.ts

```diff
@@ -17,7 +17,7 @@ export function constructFormUrl(publicKey: string) {
   if (!publicKey) {
     throw Error("public key is required");
   }
-  return `https://formstr.app/#/forms/${publicKey}`;
+  return `https://formstr.app/#/v1/fill/${publicKey}`;
```

Comment:
use ROUTES config object

Reply:
don't want to create a circular dependency, sdk should not use web-app, we can move this to a common config, or we can just leave this be.